test/unit/tag_test.rb

```diff
@@ -92,4 +92,29 @@ class TagTest < ActiveSupport::TestCase
     assert !node(:blog).responses('replication').empty?
     assert node(:blog).response_count('replication') > 0
   end
+
+  test "returns empty array if users are  following both the given tags and this tag" do
+    tag = tags(:spam)
+    given_tags = [tags(:chapter)]
+    assert_equal [], tag.followers_who_dont_follow_tags(given_tags)
+  end
+
+  test " returns users following this tags but not given tags" do
+    # users following tag are bob, unbanned_spammer, admin
+    # users following tag1 are bob, ubanned_spammer, moderator
+    # users following tag2 are spammer, newcomer
+    tag = tags(:test)
+    tag1 = tags(:awesome)
```

Comment:
Here we need to use a comment to mark out the tags where `following == false` and we should add another assertion that:

```ruby
assert !tag.followers_who_dont_follow_tags(given_tags).include?(tag_not_followed)
```

That way we're sure the `following = false` tag_selections are not showing up, and the test is readable. Make sense? 

Thanks!